plugins/lsp/lsp-packages.nix

```diff
@@ -237,6 +236,7 @@
     fsautocomplete = "fsautocomplete";
     fstar = "fstar";
     futhark_lsp = "futhark";
+    gdscript = "gdtoolkit_4";
```

Comment:
Actually, I'm not sure this is correct. GD Toolkit doesn't mention a language server.

Perhaps the correct package is `godot_4`?